tailwarden/komiser
# Feature/786 onboarding wizard

## Problem

We are seeking to complete the second portion of the onboarding wizard, This portion exists when the user has gone through the initial portion of the Onboarding wizard flow and this portion of the wizard helps in managing the existing cloud accounts and also adding newer cloud accounts.

## Solution

[Provide a detailed explanation of your proposed solution, including any relevant code snippets or screenshots.]

## Changes Made

- Added TODO to portions of the application that would be updated when Onboarding Wizard is complete
- Added changes that serve the core portion of the onboarding wizard cloud account management console.

## How to Test

Edit the beta flags in these portions of the application:

dashboard/components/layout/Layout.tsx: change `const betaFlagOnboardingWizard = false;` to `const betaFlagOnboardingWizard = true;`. This should be on `line 40` of the file

dashboard/components/layout/hooks/useGlobalStats.tsx: change `setHasNoAccounts(true);` to `setHasNoAccounts(false);`. This should be on `line 34` of the file

## Screenshots

![Screenshot 2023-09-12 at 12 54 13 PM (2)](https://github.com/tailwarden/komiser/assets/61082329/cdb0cb82-2bd6-4e35-9083-9117165d6e8e)

![Screenshot 2023-09-12 at 12 54 19 PM (2)](https://github.com/tailwarden/komiser/assets/61082329/f30b9723-6407-48d6-92e6-ef767799b815)


## Notes

[Any additional notes or information that you would like to share with the reviewers.]

## Checklist


## Reviewers

@[username of the reviewer]
